cg/clients/freshdesk/utils.py

```diff
@@ -25,7 +25,16 @@ def wrapper(*args, **kwargs):
         try:
             return func(*args, **kwargs)
         except (MissingSchema, HTTPError, ConnectionError) as error:
-            LOG.error(f"Request to Freshdesk failed: {error}")
+            if isinstance(error, HTTPError) and error.response is not None:
```

Comment:
If there is logic specifically for `HTTPError`, just move that that to a separate line.